grizzly/prometheus.libsonnet

```diff
@@ -27,4 +29,11 @@ local alertRules = 'prometheusAlerts';
       )
     for key in std.objectFields(mixins)
   },
+
+  rule_group: {
+    new(namespace, name, group)::
+      resource.new('PrometheusRuleGroup', name)
+      + resource.addMetadata('namespace', namespace)
+      + resource.withSpec(group),
```

Comment:
I'd suggest layering `group` one step deeper so you can add siblings to the group in future revisions:

```
spec:
  group: ...
```

The same for all other resources.